passthrough_test.go

```diff
@@ -270,6 +270,40 @@ func TestPassthroughBackend_Revoke(t *testing.T) {
 	test(b)
 }
 
+func TestPassthroughBackend_Renew(t *testing.T) {
+	b := testPassthroughLeasedBackend()
+
+	req := logical.TestRequest(t, logical.CreateOperation, "foo")
+	req.Data = map[string]interface{}{
+		"ttl":     "4h",
+		"payload": "alpha",
+	}
+	storage := req.Storage
+	if _, err := b.HandleRequest(context.Background(), req); err != nil {
+		t.Fatalf("err: %v", err)
+	}
+
+	req = logical.TestRequest(t, logical.RenewOperation, "foo")
+	req.Storage = storage
+	req.Secret = &logical.Secret{
+		InternalData: map[string]interface{}{
+			"secret_type": "kv",
+		},
+	}
```

Comment:
Note to reviewer: This is a bit fake, hacking together just enough of what a real RenewOperation request would look like. Ideally I'd like to run a real Core and generate a real lease renewal, but the relevant test helpers are not exposed for out-of-tree plugins to use. This is probably good enough, I guess.